prow/cmd/jenkins-operator/main.go

```diff
@@ -94,8 +102,19 @@ func main() {
 	} else {
 		logger.Fatal("An auth token for basic or bearer token auth must be supplied.")
 	}
+	var tlsConfig *tls.Config
+	if *certFile != "" && *keyFile != "" {
```

Comment:
seems like we should just validate that if any of the three (cert, key, cacert) are present that all three are?